Mathlib/Tactic/CategoryTheory/Coherence.lean

```diff
@@ -183,7 +196,7 @@ noncomputable def monoidalIsoComp {W X Y Z : C} [LiftObj X] [LiftObj Y]
   f ≪≫ asIso MonoidalCoherence.hom ≪≫ g
 
 @[inherit_doc Mathlib.Tactic.Coherence.monoidalIsoComp]
-scoped[CategoryTheory.MonoidalCategory] infixr:80 " ≪⊗≫ " =>
+scoped[CategoryTheory.MonoidalCategory] infixr:79 " ≪⊗≫ " =>
   Mathlib.Tactic.Coherence.monoidalIsoComp -- type as \ll \ot \gg
```

Comment:
Can you make a separate PR with these precedence changes, so that it's clear what the fallout is?

Reply:
I canceled the change of precedence. Setting `79`, which is lower than `80` for the usual composition `≫`, only affects one pair of parenthesis in https://github.com/leanprover-community/mathlib4/pull/6307/commits/c1bce84c5ba2714d3215f03485a3f410211fef0a. Now I don't think we really need to set a lower precedence here.